### PR TITLE
INJIMOB-1408

### PR DESCRIPTION
**Pull request by santhoshsunder**
## Description

> Describe the Changes in the PR

## Issue ticket number and link

> Example : [INJIMOB-000](https://mosip.atlassian.net/browse/INJIMOB-000&#41)

## Screenshots

> Add the Screenshots of the feature.

### CODE REVIEW

File: injitest/logs/browserstack-javaagent.log [hunk 1]
```diff
@@ -0,0 +1,38 @@
+19:22:11.529 [main] INFO  c.browserstack.BrowserStackJavaAgent - Selenium version detected - 4.13.0
```
Comment (kamalsinghthoughtworks) [inline injitest/logs/browserstack-javaagent.log:1]: add log files in git ignore and avoid to commit them.

File: injitest/src/main/java/inji/pages/SunbirdLoginPage.java [hunk 2]
```diff
@@ -146,6 +146,10 @@ public class SunbirdLoginPage extends BasePage {
     @iOSXCUITFindBy(accessibility = "benefitsValue")
     private WebElement benefitsValue;
 
+    @iOSXCUITFindBy(xpath = "//XCUIElementTypeStaticText[@name=\"Authentication Failed:\"]")
+    @AndroidFindBy(xpath  = "//android.widget.TextView[@text=\"Authentication Failed: \"]")
+    private WebElement invalidMessage;
```
Comment (kamalsinghthoughtworks) [inline injitest/src/main/java/inji/pages/SunbirdLoginPage.java:151]: Can we use accessibility id for android and iOS both?

File: injitest/src/main/java/inji/pages/SunbirdLoginPage.java [hunk 3]
```diff
@@ -167,6 +171,23 @@ public void enterPolicyNumberTextBox(String PolicyNo) {
         sendKeysToTextBox(enterPolicyTextBox, PolicyNo);
     }
 
+    public void enterInvalidPolicyNumberTextBox(String PolicyInvalidNo) {
+        try {
+            Thread.sleep(1000);
```
Comment (kamalsinghthoughtworks) [inline injitest/src/main/java/inji/pages/SunbirdLoginPage.java:176]: why we need hard wait ?

File: injitest/src/main/java/inji/pages/SunbirdLoginPage.java [hunk 4]
```diff
@@ -176,6 +197,15 @@ public void enterFullNameTextBox(String fullname) {
             sendKeysToTextBox(enterFullnameTextBox, fullname);
     }
 
+    public void enterInvalidFullNameTextBox(String fullname) {
+        try {
+            Thread.sleep(1000);
```
Comment (kamalsinghthoughtworks) [inline injitest/src/main/java/inji/pages/SunbirdLoginPage.java:202]: why we need hard wait ?

**Comment by swatigoel** [comment]
@santhoshsunder why example link in PR is [INJIMOB-000]

**Pull request closed without merging**